syllabus/6-Polkadot-SDK/Polkadot-SDK-slides.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Introduction to Polkadot SDK
+description: In Introduction to Polkadot SDK
+duration: 120 mins
+---
+
+# Introduction to Polkadot SDK
+
+Note:
+
+This lecture is about learning two things:
+
+1. Polkadot
+2. Polkadot SDK
+   1. Substrate
+   2. FRAME
+   3. XCM
+
+There will be amazing modules going into further details of each. This will be very high level.
+
+---
+
+## Learning Outcomes
+
+1. Understand what Polkadot is as a developer platform.
+2. Understand the pieces of the Software Development Kit (SDK) of this platform
+
+---
+
+## Polkadot and its SDK
+
+---
+
+### 1. Polkadot
+
+Polkadot stems from Ethereum. A single threaded smart contract blockchain.
+
+Let's start with Ethereum.
+
+Let's make it sharded, while sharing security. Execution sharding, the holy grain of what ETH 2.0
+was meant to be.
+
+Let's make it heterogenous. The state transition function of these shards need NOT be the same. WASM
+is used.
+
+Let's make the deployment of those shards flexible and permissionless. We arrive at Polkadot what it
+is today.
+
+> Polkadot 1.0 is a platform to launch blockchains that can progress in parallel, whilst sharing the
+> security of Polkadot.
+
+In this architecture, we call Polkadot the Relay Chain. we call each of these blockchains a
+Parachain.
```

Comment:
I'd be cautious here. It is to this day confusing to me and different entities view it differently so I'd be careful about opinionation. Polkadot is often used to describe the whole system (relay chain + parachains). I wouldn't go as far as saying that Polkadot is only the relay chain.

The heart of Polkadot is the relay chain that aggregates and validates the state of all the small paralell blockchains - parachains.

Reply:
Right, my wording is kinda bad here. I meant to say that: 

the base blockchain is called `Polkadot Relay Chain`, the parallelized blockchains connected are called `Parachains`. I would indeed avoid describing what "Polkadot" as a general term mean.